icons/new/attention.svg

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If anyone was using this icon, this would mean a breaking change – did you rename this one?

Reply:
I think I removed it by accident :/